apps/ledger-live-desktop/src/renderer/init.jsx

```diff
@@ -132,6 +133,13 @@ async function init() {
 
   const isMainWindow = remote.getCurrentWindow().name === "MainWindow";
 
+  // TODO hydrate the store with the bridge/cache
+  listCachedCurrencyIds().then(ids => {
+    ids.forEach(id => {
+      hydrateCurrency(id);
+    });
+  });
+
```

Comment:
```suggestion
  // hydrate the store with the bridge/cache
  await Promise.allSettled(
    listCachedCurrencyIds()
      .map(id => {
        const currency = findCryptoCurrencyById(id);
        return currency ? hydrateCurrency(currency) : null;
      }),
  );
```

Reply:
`listCachedCurrencyIds()` is not async, but `hydrateCurrency` is, so here is the working version. 
Also missing an import of `findCryptoCurrencyById` for my suggestion to work

Reply:
ooooh